exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/transactions/UnknownTx.java

```diff
@@ -34,32 +33,27 @@
  * once they receive it as a message from this node. If multiple unknown transaction messages
  * need to be submitted, a seed might be added.
  */
-public final class UnknownTx extends AbstractTransaction {
+public final class UnknownTx implements Transaction {
 
-  static final short ID = 9999;
-
-  public UnknownTx() {
-    super(createMessage());
-  }
-
-  @Override
-  public boolean isValid() {
-    return true;
-  }
+  private static final short ID = 9999;
 
   @Override
-  public void execute(Fork view) {
+  public void execute(TransactionContext context) {
     throw new AssertionError("Must never be executed by the framework: " + this);
   }
 
-  private static BinaryMessage createMessage() {
-    return new Message.Builder()
-        .setServiceId(QaService.ID)
-        .setMessageType(ID)
-        .setNetworkId((byte) 0)
-        .setVersion((byte) 0)
-        .setBody(ByteBuffer.allocate(0))
-        .setSignature(new byte[Message.SIGNATURE_SIZE])
-        .buildRaw();
+  /**
+   * Returns raw transaction.
+   */
+  public static RawTransaction createRawTransaction() {
+    byte[] seed = new byte[1];
```

Comment:
I'd not create a random seed at the service side, especially such a small one. Unless it is required for ITs, I'd keep using an empty body, allowing only a single such transaction.